app/extensions/edm/__init__.py

```diff
@@ -109,7 +118,7 @@ def _endpoint_tag_fmtstr(self, tag):
 
 class EDMManagerUserMixin(object):
     def check_user_login(self, username, password):
-        self.ensure_initialed()
+        self._ensure_initialed()
 
```

Comment:
btw, in my **enc_tests** branch i have _fixed this typo_ -- changed it to `initialized` cuz i was tired of typing it wrong, heh.

Reply:
Ha, lets get that change in! I was a little confused by the name also.